tests/rest/client/test_third_party_rules.py

```diff
@@ -49,7 +49,7 @@ class ThirdPartyRulesTestCase(unittest.HomeserverTestCase):
     def make_homeserver(self, reactor, clock):
```

Comment:
in case it's not obvious: this file is being renamed, so that the tests actually get run.